example/custom_error_example/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/epsagon/epsagon-go/epsagon"
+)
+
+func doTask(a int, b string) (int, error) {
+	log.Printf("inside doTask: b = %s", b)
+	if a < 5 {
```

Comment:
for the example I'd put a condition that's met, so someone that runs it will see the error